OpenRA.Mods.Common/Widgets/Logic/Ingame/LoadIngamePlayerOrObserverUILogic.cs

```diff
@@ -53,6 +54,12 @@ public LoadIngamePlayerOrObserverUILogic(Widget widget, World world)
 				};
 			}
 
+			var console = world.WorldActor.TraitOrDefault<ChatCommands>();
+			var help = world.WorldActor.TraitOrDefault<HelpCommand>();
+
+			if (console != null && help != null)
```

Comment:
Why does this depend on the chat and help commands? The geometry overlay can be enabled through the debug menu too!